scripts/pre_commit_linter.py

```diff
@@ -1312,6 +1336,84 @@ def _check_js_component_name_and_count(self):
             print ''
             return summary_messages
 
+    def check_for_mandatory_pattern_in_file(
+            self, pattern_set, filepath, failed):
+        """Checks for a given mandatory pattern in a file.
```

Comment:
The docstring seems to be wrong. The function checks a given list of patterns in a particular file.

Reply:
Corrected!